approved-books.json

```diff
@@ -374,5 +702,13 @@
     "version": "1.9.1",
     "server": "cnx.org",
     "uuid": "e42bd376-624b-4c0f-972f-e0c57998e765"
+  }
```

Comment:
@theresamasciale - Ah, I think the CI is failing to parse the file as JSON because this line needs a comma at the end.

Reply:
oo ok, is there a way to edit my changes so that I dont have to start all over again?

Reply:
jk i figured it out. how about now?